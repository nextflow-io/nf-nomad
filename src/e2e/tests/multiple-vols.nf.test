nextflow_workflow {

    name "Test Multiple vols"
    script "workflows/multiple-vols.nf"
    config "tests/multiple-vols.config"
    workflow "MAIN"

    test("Should run without failures") {

        when {
            params {
            }
            workflow {
            }
        }

        then {
            assert workflow.success
            assert snapshot(workflow.out).match()
        }

    }

}
