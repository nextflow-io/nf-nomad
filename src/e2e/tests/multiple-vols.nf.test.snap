{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "Bonjour\n",
                    "Ciao\n",
                    "Hello\n",
                    "Hola\n"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.3",
            "nextflow": "25.04.7"
        },
        "timestamp": "2025-10-20T20:44:12.406843585"
    }
}